examples/online-compile/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_executable(shaderc-online-compile main.cc)
+shaderc_default_compile_options(shaderc-online-compile)
+target_include_directories(shaderc-online-compile PRIVATE ${libshaderc_SOURCE_DIR}/include)
```

Comment:
Just a minor thought, I guess we don't need to add libshaderc's include directory explicitly. The include directory should be carried with the library.